src/main/java/com/conveyal/r5/point_to_point/PointToPointRouterServer.java

```diff
@@ -113,6 +121,8 @@ private static void run(TransportNetwork transportNetwork) {
         ObjectReader graphQlRequestReader = mapper.reader(GraphQlRequest.class);
         ObjectReader mapReader = mapper.reader(HashMap.class);
         staticFileLocation("debug-plan");
+        transportNetwork.transitLayer.buildDistanceTables(null);
```

Comment:
Generally we don't need to build these tables in point-to-point routing, so this may be adding a lot of startup time and memory consumption when someone just wants to do routing.